repository/kafka/docs/latest/configuration.md

```diff
@@ -75,7 +75,7 @@ You can install the KUDO Zookeeper or you can use any other Zookeeper cluster yo
 You can configure KUDO Kafka to use Zookeeper using the parameter `ZOOKEEPER_URI`
 Let's see this with an example:
 ```
-kubectl kudo install kafka --instance=my-kafka-cluster \
+$ kubectl kudo install kafka --instance=my-kafka-cluster \
```

Comment:
```suggestion
$ kubectl kudo install kafka --instance=my-kafka-name \
```

Reply:
let's use `my-kafka-name` to be consistent everywhere

Reply:
nice catch!